packages/datadog-instrumentations/src/mysql.js

```diff
@@ -1,10 +1,6 @@
 'use strict'
 
-const {
-  channel,
-  addHook,
-  AsyncResource
-} = require('./helpers/instrument')
+const { channel, addHook, AsyncResource } = require('./helpers/instrument')
```

Comment:
Is the remaining usage of `AsyncResource` in this file intentional?

Reply:
They aren't, see #5840